packages/debugger/lib/evm/actions/index.js

```diff
@@ -1,34 +1,8 @@
 export const ADD_CONTEXT = "EVM_ADD_CONTEXT";
-export function addContext({
-  context,
-  contractName,
-  binary,
-  sourceMap,
-  primarySource,
-  immutableReferences,
-  compiler,
-  compilationId,
-  abi,
-  contractId,
-  contractKind,
-  isConstructor,
-  linearizedBaseContracts
-}) {
+export function addContext(context) {
```

Comment:
Would you consider `createContextAction` as this seems like an action creator?

Reply:
Yeah, I don't really want to go renaming this.  Also, all of our actions work like this?  Like, I'd have to rename every single action function to `createEtcAction`, and then use those names everywhere... that would be so wordy!